infra/content/kXml.class.php

```diff
@@ -27,6 +27,12 @@ public function errorHandler($errNo, $errStr, $errFile, $errLine)
 	}
 }
 
+function xml_load_for_xslt($xmlStr)
```

Comment:
Did you try and manipulate http://www.w3schools.com/xsl/func_document.asp to work ? instead of calling PHP